profile_change_goal_fragment.xml

```diff
@@ -0,0 +1,214 @@
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/profile_location_header" />
+
+    <TextView
+        android:id="@+id/here_to_request_test"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="193dp"
+        android:lineSpacingExtra="-2sp"
+        android:text="I am here to request: "
+        android:textColor="@color/color_primary_brand_color"
+        android:textSize="20sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/profile_location_header" />
+
+    <Button
+        android:id="@+id/donation_save_btn"
+        style="@style/PrimaryButton"
+        android:layout_width="312dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="402dp"
+        android:lineSpacingExtra="5.9sp"
+        android:text="@string/save"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/profile_location_header" />
+
+    <TextView
+        android:id="@+id/information_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginBottom="16dp"
+        android:lineSpacingExtra="5.6sp"
+        android:text="@string/information"
+        android:textColor="@color/color_primary_text"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toTopOf="@+id/volunteerHR_text"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/donation_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="14dp"
+        android:layout_marginBottom="16dp"
+        android:lineSpacingExtra="5.6sp"
+        android:textColor="@color/color_primary_text"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toTopOf="@+id/information_text"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/Here_to_offer_text"
+        tools:text="Donations" />
+
+    <TextView
+        android:id="@+id/volunteerHR_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="20dp"
+        android:layout_marginBottom="34dp"
+        android:lineSpacingExtra="5.6sp"
+        android:text="@string/volunteer_hours"
+        android:textColor="@color/color_primary_text"
+        android:textSize="14sp"
+        app:layout_constraintBottom_toTopOf="@+id/here_to_request_test"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <ImageView
+        android:id="@+id/profile_donation_back_arrow"

```

Comment:
We already have a file with the AppBar layout you just need to include the file like this.

![image](https://user-images.githubusercontent.com/6819641/95862447-51b44e00-0d39-11eb-960a-880a4db13f2c.png)